src/components/ProjectAuditTable.tsx

```diff
@@ -24,6 +24,7 @@ interface Props {
   searchField: string;
   searchValue: string;
   setAudit: (audit: AuditAction[]) => void;
+  setProductsNavbarImageData: (imageName: ImageData) => void;
```

Comment:
Make this prop optional and set a default or change the PR's title to "feat!: Added..." to make it a breaking change?

Reply:
and perhaps also rename `imageName` as `imageData`?